web/impact/impact/v1/views/reserve_office_hour_view.py

```diff
@@ -172,17 +177,22 @@ def prepare_email_notification(self,
             startup_name = self.startup.organization.name
         else:
             startup_name = ""
-
+        calendar_data = self.get_calendar_data(counterpart)
         start_time = localized_office_hour_start_time(self.office_hour)
         context = {"recipient": recipient,
                    "counterpart": counterpart,
                    "office_hour_date_time": start_time,
                    "startup": startup_name,
-                   "message": self.message}
+                   "message": self.message,
+                   "calendar_data": calendar_data
+                   }
+        context.update(generate_calendar_links(self.hour))
         body = loader.render_to_string(template_path, context)
         return {"to": [recipient.email],
                 "subject": self.SUBJECT,
-                "body": body}
+                "body": body,
+                "attachment": (ICS_FILENAME, calendar_data['ical_content'],
```

Comment:
one parameter per line - please drop `calendar_data['ical_content']` to next line